spec/unit/export_task_spec.rb

```diff
@@ -25,11 +25,19 @@
     double(
       name: "kiwi",
       export_name: "name-type",
-      system_description: create_test_description(scopes: ["os", "unmanaged_files"])
+      system_description: create_test_description({scopes: ["os"], extracted_scopes: ["unmanaged_files"]})
```

Comment:
Line is too long. [106/100]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.